script/build

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+
+if [[ -z $SERVICE ]]; then
+  SERVICE=${PWD##*/}
+fi
```

Comment:
Or would getting it from the `CONTAINER` env var in the .service file be better?